integration/bundle/testdata/default_python/bundle_deploy.txt

```diff
@@ -0,0 +1,6 @@
+Building project_name_$UNIQUE_PRJ...
```

Comment:
What is benefit of using txt files with custom syntax replacements instead of Go templates?

Reply:
These files are not templates - I don't generate anything from them.